src/frontend/components/Players/PaginationNav.tsx

```diff
@@ -0,0 +1,73 @@
+import { FC } from "react";
+import { cn } from "~frontend/utils";
+
+const Search: FC<{ className?: string }> = ({ className }) => {
```

Comment:
pagination component not search (confusing) - needs to pass pages and onClick event as props